plugins/playlist/src/components/PlaylistCard/PlaylistCard.tsx

```diff
@@ -108,7 +108,7 @@ export const PlaylistCard = ({ playlist }: PlaylistCardProps) => {
           <Typography variant="body2" className={classes.label}>
             Description
           </Typography>
-          <MarkdownContent content={playlist.description ?? ''} />
+          <OverflowTooltip text={playlist.description ?? ''} line={4} />
```

Comment:
I don't think we want to replace `MarkdownContent` here, this would remove support for using Markdown in your description - I suggest adding a scroll when the content overflows instead